express-delete/index.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+const app = express();
+
+// console.log(grades)
+
+app.delete('/api/grades/:id', (req, res) => {
+
+  delete grades['12'];
```

Comment:
You don't want to hard code the `id` here. You should be using `req.params.id` to get the `id` to delete